docs/remove_rhts.rst

```diff
@@ -0,0 +1,278 @@
++------------------------+----------------------------------------+
+| Requires               | [restraint] dependencies               |
++------------------------+----------------------------------------+
+| RhtsRequires           | [restraint] dependencies               |
++------------------------+----------------------------------------+
+| USE_PTY                | [restraint] use_pty                    |
++------------------------+----------------------------------------+
+
+The following are informational variables and should be maintained.
+`Restraint` does not perform any action on these variables.
+
++------------------------+----------------------------------------+
+| testinfo.desc variable | metadata [section] variable substitute |
++========================+========================================+
+| License                | [General] license                      |
++------------------------+----------------------------------------+
+| Owner                  | [General] owner                        |
++------------------------+----------------------------------------+
+| Description            | [General] description                  |
++------------------------+----------------------------------------+
+| Confidential           | [General] confidential                 |
++------------------------+----------------------------------------+
+| Destructive            | [General] destructive                  |
++------------------------+----------------------------------------+
+
+There are no substitutes for the following `Makefile/testinfo.desc` variables
+in `Restraint's` metadata file.  Some of these variables are
+informational and can be added in the metadata file but it is just
+documentation.  `Restraint` will not act on them and they will be ignored.
+
+* TESTVERSION
+* FILES
+* BUILT_FILES
+* TEST_DIR
+* Path
+* Architectures
+* Bugs
+* Priority
+* Releases
+* RhtsOptions
+* TestVersion
+
+Example of removing testinfo.desc file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The sample files below show converting `Makefile/testinfo.desc` to `metadata` file.
+The Makefile does not have to be removed in its entirety. In the Sample Makefile,
+everything from `rhts-make.include` below should be removed.  If the
+upper part of the Makefile is kept, the `entry_point` variable defined in the `metadata`
+file is not required since `Restraint` will perform `make run` when `entry_point` is
+not present.
+
+Sample Makefile::
+
+ export TEST=/examples/no-rhts/sample-before
+ export TESTVERSION=1.0
+
+ BUILT_FILES=
+
+ FILES=$(METADATA) runtest.sh Makefile PURPOSE
+
+ .PHONY: all install download clean
+
+ run: $(FILES) build
+         ./runtest.sh
+
+ build: $(BUILT_FILES)
+         test -x runtest.sh || chmod a+x runtest.sh
+
+ clean:
+         rm -f *~ $(BUILT_FILES)
+
+ include /usr/share/rhts/lib/rhts-make.include
+
+ $(METADATA): Makefile
+         @echo "Owner:           Happy Golucky <hgolucky@funland.com>" > $(METADATA)
```

Comment:
use example or something that doesn't exist

Reply:
done